config/serverless.yml

```diff
@@ -6,6 +6,14 @@ xpack.fleet.internal.fleetServerStandalone: true
 xpack.fleet.internal.disableILMPolicies: true
 xpack.fleet.internal.disableProxies: true
 
+# Enable ZDT migration algorithm
+migrations.algorithm: zdt
+
+# temporarily allow to run the migration on UI nodes
+migrations.zdt:
+  metaPickupSyncDelaySec: 5
+  runOnRoles: ["ui"]
```

Comment:
The temporary workaround until we're able to run the migrator node during upgrades

Reply:
Can we add what the dependency is here? In other words, a phrase similar to "until the Kibana Controler is ready to take over the job" with a link to the work Michael's doing.

Reply:
> with a link to the work Michael's doing.

I wanted to, but it's a private repo, so didn't feel like linking it here